src/apps/startups/migrations/0005_project_startup_projects.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.10 on 2024-02-21 15:47
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('startups', '0004_startup_bio'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Project',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('project_name', models.CharField(max_length=64, verbose_name='Project Name')),
+                ('type', models.CharField(blank=True, choices=[('P', 'Project'), ('I', 'Idea')], default='S', help_text='Project type', max_length=1)),
+                ('created_at', models.DateTimeField()),
```

Comment:
Consider using ```auto_now_add=True``` to automatically set this field to the current date and time when a project instance is created

Reply:
Fixed in 300042ae1be1b767a40a218997f6a254c08f2771